jerry-main/main-unix.c

```diff
@@ -354,6 +356,10 @@ main (int argc,
   bool is_save_snapshot_mode_for_global_or_eval = false;
   const char *save_snapshot_file_name_p = NULL;
 
+  bool is_save_literals_mode = false;
+  bool is_save_literals_mode_in_c_format_or_list = false;
+  const char *save_literals_file_name_p = NULL;
```

Comment:
Question: can snapshot and save literals be done in one step? It is great if it is possible. Othoerwise only one file name parameter is enough.